watershed-android/watershed/app/src/main/java/com/blueprint/watershed/Users/UserFragment.java

```diff
@@ -88,9 +88,10 @@ private void initializeListLayout() {
         ArrayList<String> options = new ArrayList <String>();
 
         options.add("Field-Reports " + mUser.getFieldReportsCount());
+        options.add("Tasks " + mUser.getFieldReportsCount());
+        options.add("Sites " + mUser.getSitesCount());
         if (mUser.isEmployee() || mUser.isManager()){
```

Comment:
They aren't (except for Field report count I think), do we want this information to even be available though? Sites and Tasks we can access already. I may just take them out.